client/src/components/TrackGroup/TrackGroupEmbed.tsx

```diff
@@ -67,7 +67,7 @@ const TrackGroupEmbed: React.FC<{
             onClick={() => {
               if (trackGroup.artist) {
                 navigator.clipboard.writeText(
-                  process.env.REACT_APP_CLIENT_DOMAIN +
+                  import.meta.env.VITE_CLIENT_DOMAIN +
```

Comment:
Just wondering out loud: would it be better to ultimately replace these with something like window.location? Is that naive?

Reply:
That would make sense to me! I've found the [`URL(url, base)`](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) constructor useful for similar logic in the past, as it ensures the result is a valid link - so this could be `new URL(getReleaseUrl(...), window.location.origin).toString()`